python/requirements_compiled.txt

```diff
@@ -1205,7 +1205,7 @@ numpy==1.26.4
     #   onnx
     #   onnxruntime
     #   open-spiel
-    #   opencv-python
+    #   opencv-python-headless
```

Comment:
could you separate the change of this file into another PR? does not seem to be related to this change.

Reply:
Sure. I'll see if it is possible to diff the generated file with the codebase to make sure this file is changed upon any dependencies change.